### DISCUSSION TITLE
Where get monitoring data?

### DISCUSSION THREAD

**Discussion by orlovds** [OpenIdentityPlatform/OpenIG#96] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
How to get monitoring metrics?

**Describe the solution you'd like**
You need a point for getting metrics in prometeus format

**Describe alternatives you've considered**
I don't see any alternatives

**Additional context**
Metrics on connections, load, errors are needed

**Comment by vharseko** [comment, marked as answer]
[Auditing and Monitoring OpenIG](https://doc.openidentityplatform.org/openig/gateway-guide/chap-auditing)

**Comment by orlovds** [comment]
@vharseko Thank you very much!
